stdlib/std.jsonnet

```diff
@@ -136,6 +136,22 @@ limitations under the License.
     else
       replace_after(0, 0, ''),
 
+  asciiUpper(x)::
+    local cp = std.codepoint;
+    local up_letter(c) = if cp(c) >= 97 && cp(c) < 123 then
+      std.char(cp(c) - 32)
+    else
+      c;
+    std.join('', std.map(up_letter, std.stringChars(x))),
+
+  asciiLower(x)::
+    local cp = std.codepoint;
+    local down_letter(c) = if cp(c) >= 65 && cp(c) < 91 then
+      std.char(cp(c) + 32)
+    else
```

Comment:
nit: the indentation looks a bit weird here, it looks as if `else` matched `local`

Reply:
It's what the formatter does, and I think it's ok because it's like

```
local x = [
  0,
];
```